### PR TITLE
✨ Add Sequencing experiment model with tests

### PR DESCRIPTION
**Pull request by parimalak**
Add Sequencing experiment model with tests

### CODE REVIEW

File: dataservice/api/aliquot/models.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    """
+    __tablename__ = "aliquot"
+    external_id = db.Column(db.Text())
+    shipment_origin = db.Column(db.Text())
+    shipment_destination = db.Column(db.Text())
+    analyte_type = db.Column(db.Text(), nullable=False)
+    concentration = db.Column(db.Integer())
+    volume = db.Column(db.Float())
+    shipment_date = db.Column(db.DateTime())
+    sequencingexperiments = db.relationship(SequencingExperiment,
```
Comment (znatty22) [inline dataservice/api/aliquot/models.py]: Use underscore to separate words:  `sequencing_experiments`

File: dataservice/api/aliquot/models.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+from dataservice.extensions import db
+from dataservice.api.common.model import Base
+from dataservice.api.sequencingexperiment.models import SequencingExperiment
```
Comment (znatty22) [inline dataservice/api/aliquot/models.py]: Use underscore to separate words: `sequencing_experiment`

File: dataservice/api/sequencingexperiment/models.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    :param platform: Name of the platform used to obtain data
+    :param instrument_model:Text term that represents the model of instrument
+    :param max_insert_size: Maximum size of the fragmented DNA
+    :param mean_insert_size: Mean size of the fragmented DNA
+    :param mean_depth:(Coverage)Describes the amount of sequence data that
+           is available per position in the sequenced genome territory
+    :param total_reads: total reads
+    :param mean_read_length: mean lenth of the reads
+    """
+    __tablename__ = "sequencingexperiment"
```
Comment (znatty22) [inline dataservice/api/sequencingexperiment/models.py]: Use underscore to separate words:  `sequencing_experiment`

File: tests/aliquot/test_aliquot_models.py [hunk 4]
```diff
@@ -0,0 +1,463 @@
+from datetime import datetime
```
Comment (znatty22) [inline tests/aliquot/test_aliquot_models.py]: Isn't this file already part of the aliquot-model branch which has to be merged in?

**Review by znatty22 — CHANGES_REQUESTED** [review]

File: tests/aliquot/test_aliquot_models.py (hunk 4 above)
Comment (dankolbman) [reply tests/aliquot/test_aliquot_models.py]: should be taken care of after a rebase

**Comment by dankolbman** [comment]
Can you rebase on master? It's difficult to tell what is part of this feature.

**Review by znatty22 — APPROVED** [review]
👍

**Review by dankolbman — APPROVED** [review]
nice! 🎉

**Pull request merged**